src/getClasses.js

```diff
@@ -6,29 +6,22 @@
  */
 export function getClasses( el )
 {
-  let classNames;
-
-  try
+  if( !el.hasAttribute( 'class' ) )
   {
-    classNames = el.classList.toString().split( ' ' );
+    return [];
   }
-  catch ( e )
-  {
-    if( !el.hasAttribute( 'class' ) )
-    {
-      return [];
-    }
 
+  try {
+    return Array.from(el.classList);
```

Comment:
I don't know what this library is supposed to support, but keep in mind that `Array.from` is absent in some older browsers: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility